docs/modules/ROOT/pages/how-tos/vsphere/install.adoc

```diff
@@ -0,0 +1,189 @@
+= Install OpenShift 4 on vSphere
+:ocp-minor-version: 4.11
+:k8s-minor-version: 1.24
+:ocp-patch-version: {ocp-minor-version}.9
+:provider: vsphere
+
+[abstract]
+--
+Steps to install an OpenShift 4 cluster on VMWare vSphere.
+
+These steps follow the https://docs.openshift.com/container-platform/latest/installing/installing_vsphere/installing-vsphere-installer-provisioned-customizations.html[Installing a cluster on vSphere] docs to set up an installer provisioned installation (IPI).
+--
+
+[IMPORTANT]
+--
+This how-to guide is an early draft.
+So far, we've setup only one cluster using the instructions in this guide.
+--
+
+[NOTE]
+--
+The certificates created during bootstrap are only valid for 24h.
+So make sure you complete these steps within 24h.
+--
+
+== Starting situation
+
+* You already have a Project Syn Tenant and its Git repository
+* You have a CCSP Red Hat login and are logged into https://console.redhat.com/openshift/install/vsphere/installer-provisioned[Red Hat Openshift Cluster Manager]
++
+IMPORTANT: Don't use your personal account to login to the cluster manager for installation.
+* You have credentials for the target vSphere cluster with the permissions described in the https://docs.openshift.com/container-platform/4.12/installing/installing_vsphere/installing-vsphere-installer-provisioned.html#installation-vsphere-installer-infra-requirements-account_installing-vsphere-installer-provisioned[upstream documentation].
+* You want to register a new cluster in Lieutenant and are about to install Openshift 4 on vSphere
+
+== Prerequisites
+
+include::partial$/install/prerequisites.adoc[]
+* `unzip`
+
+== Cluster Installation
+
+include::partial$install/register.adoc[]
+
+=== Configure input
+
+.vSphere API
+[source,bash]
+----
+export VCENTER_HOSTNAME=<vcenter hostname> <1>
+export VCENTER_API_URL="https://${VCENTER_HOSTNAME}"
+----
+<1> The vCenter hostname must be provided without the leading `https://`.
+
+.vSphere credentials
+[source,bash]
+----
+export VSPHERE_USERNAME=<username>
+export VSPHERE_PASSWORD=<password>
+----
+
+.vSphere resources
+[source,bash]
+----
+export VSPHERE_CLUSTER=<cluster name>
+export VSPHERE_DATACENTER=<datacenter name>
+export VSPHERE_DATASTORE=<datastore name>
+export VSPHERE_FOLDER=<folder name>
```

Comment:
Is this required? Don't see it referenced anywhere else?

Reply:
It's not required by default. I've removed it for now.